docusaurus.config.js

```diff
@@ -9,7 +9,7 @@ const config = {
   title: "Kubesimplify",
   tagline: "Simplifying Cloud Native for All",
   url: "https://kubesimplify.github.io",
-  baseUrl: "/website/",
+  baseUrl: "/",
```

Comment:
I think this would break the deployment as, according to the docusaurus docs, in the baseURL should be set to the repository name of your project!
Refer this section of the docs - https://docusaurus.io/docs/deployment#configuration

Lets keep it this way for now & we can come back to this topic when we'll be hosting our website on a platform & setting up a custom domain name

Let me know your thoughts :)
cc @anubha-v-ardhan @AvineshTripathi

Reply:
After giving it a 2nd look, there are few things given related to this change in the docs! We can try implementing those steps
Ref: https://docusaurus.io/docs/deployment#deploying-to-github-pages

Reply:
According to this note (from the docs), we should be able to remove the `/` once we'll have a custom domain set up
![Screenshot 2022-05-14 at 11 50 43 AM](https://user-images.githubusercontent.com/72245772/168413538-a663b8fd-a588-425b-9650-5a7422b3c61d.png)

Reply:
>I think this would break the deployment as, according to the docusaurus docs, the baseURL should be set to the repository name of your project!

I had the same doubt on #31 but now that we are sure this may break deployment, I'm closing this PR. @Shivansh-yadav13, sorry for inconvenience, we can think about this while setting up our custom domain.